kubernetes-client/test/src/com/goyeau/kubernetes/client/operation/WatchableTests.scala

```diff
@@ -41,44 +41,29 @@ trait WatchableTests[F[_], Resource <: { def metadata: Option[ObjectMeta] }]
   def deleteResource(namespaceName: String, resourceName: String)(implicit client: KubernetesClient[F]): F[Status] =
     deleteApi(namespaceName).delete(resourceName)
 
-  private def update(namespaceName: String, resourceName: String)(implicit client: KubernetesClient[F]) =
-    for {
-      resource <- getChecked(namespaceName, resourceName)
-      status   <- createOrUpdate(namespaceName, resource)
-      _ = assertEquals(status, Status.Ok)
-    } yield ()
-
   private def createOrUpdate(namespaceName: String, resource: Resource)(implicit
       client: KubernetesClient[F]
-  ): F[Status] =
-    namespacedApi(namespaceName).createOrUpdate(modifyResource(resource))
-
-  private def sendEvents(namespace: String, resourceName: String)(implicit client: KubernetesClient[F]) =
+  ): F[Unit] =
     for {
-      _ <- retry(
-        create(namespace, resourceName),
-        maxRetries = 30,
-        actionClue = Some(s"Creating $resourceName in $namespace ns")
-      )
-      _      <- retry(update(namespace, resourceName), actionClue = Some(s"Updating $resourceName"))
-      status <- deleteResource(namespace, resourceName)
-      _ = assertEquals(status, Status.Ok, status.sanitizedReason)
+      status <- namespacedApi(namespaceName).createOrUpdate(resource)
+      _ = assertEquals(status.isSuccess, true, status.sanitizedReason)
```

Comment:
We don't check for equality with Status.Ok because it can also be Status.Created (201)